src/electron/platform/android/appium-service-configurator.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import ADB from 'appium-adb';
+import {
+    AndroidServiceConfigurator,
+    DeviceInfo,
+    PackageInfo,
+    PermissionInfo,
+} from 'electron/platform/android/android-service-configurator';
+
+type AdbDevice = {
+    udid: string;
+};
+
+const servicePackageName: string = 'com.microsoft.accessibilityinsightsforandroidservice';
+
+export class AppiumServiceConfigurator implements AndroidServiceConfigurator {
+    private readonly adb: ADB;
+
+    constructor(adb: ADB) {
```

Comment:
You can avoid some of the boilerplate by declaring the field within the ctor (`public constructor(private readonly adb: ADB) { }`)

Reply:
Nice!